Library/Homebrew/extend/ENV/std.rb

```diff
@@ -287,7 +287,7 @@ def cxx11
     if compiler == :clang
       append "CXX", "-std=c++11"
       append "CXX", "-stdlib=libc++"
-    elsif compiler =~ /gcc-(4\.(8|9)|5)/
+    elsif compiler =~ /gcc-(4\.(8|9)|5|6)/
```

Comment:
May even be worth having a method for this regex in `ENV/shared.rb` as they are the same in both places.